docs/src/modules/java/pages/workflows.adoc

```diff
@@ -6,6 +6,33 @@ include::ROOT:partial$include.adoc[]
 
 Implementing a business transaction that spans multiple services is one of the major challenges in distributed systems implementation. Fortunately, with Kalix ecosystem and concepts like https://docs.kalix.io/spring/actions-publishing-subscribing.html[Subscription], it's very easy to build an event driven choreography that covers a complex business process. A https://microservices.io/patterns/data/saga.html[Saga pattern] that generalizes this solution was never so easy to implement. However, for some use cases a different flavor of the Saga pattern, an orchestration, might be more suitable. That was the main driver behind a Kalix component type called a Workflow. From a high-level perspective it joins the power of https://docs.kalix.io/concepts/state-model.html[Kalix Entities state model] (durability, consistency guaranties) and https://docs.kalix.io/spring/actions.html[Kalix Actions] ability to call other components and services. You can model your business process in a single place and the Workflow will keep it running or rollback in case of a failure.
 
+
+== Identifying the Workflow
+
+In order to interact with a Workflow in Kalix, we need to assign an *type id* and one or more instance *ids*:
+
+* *type id* is a unique identifier to the workflow. To define the workflow type id, the workflow class must be annotated with `@TypeId` and have a unique and stable identifier assigned.
```

Comment:
@aludwiko, here is the point I realised that workflows are sharing the same space as ES entities in the Proxy. 

It will be better if we prefix the persistence id for workflows internally (in the proxy) so we are sure that they never conflict (until a user prefixes their own entity type id themselves with the same prefix 🤦 ).

Reply:
As discussed, a prefix and a quick deployment are possible, because I'm the only one using this entity on prod (confirmed based on metrics). On the other hand, maybe we should not prefix it and create a separate table for events from workflows. I think it's possible, but I'm not sure about the overall ops cost of such a change.

Reply:
Another table would be nicer but increases of ops cost on our side with no real benefit for users. Not only we will need to manage another table, but there are also other things to put in place, like tracking usage, billing, etc.